### PR TITLE
Feature/registers instances

### PR DESCRIPTION
**Pull request by pastorpflores**

### CODE REVIEW

File: firmware/Sources/APP/APP_SALG/app_salg.c [hunk 1]
```diff
@@ -43,6 +44,42 @@
...
+MID_REG_errorStatus_s APP_SALG_errorStatus = {
+		MID_REG_ERROR_NONE,	// hsVoltErr
+		MID_REG_ERROR_NONE,	// lsVoltErr
+		MID_REG_ERROR_NONE,	// lsCurrErr
+		MID_REG_ERROR_NONE,	// commErr
+		MID_REG_ERROR_NONE,	// tempErr
+		MID_REG_ERROR_NONE,	// intErr
+		0					// lastErrVal
+};
+MID_REG_meas_s APP_SALG_meas = {
```
Comment (mariuscrsn) [inline firmware/Sources/APP/APP_SALG/app_salg.c:74]: Change to MID_REG_data_s and use it for meas, factors and offset on EPC for calibration

**Review by mariuscrsn — CHANGES_REQUESTED** [review]

File: firmware/Sources/APP/APP_IFACE/app_iface.c [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+/**********************************************************************************/
+
+/**********************************************************************************/
+/*                        Definition of exported functions                        */
+/**********************************************************************************/
+
+/*This register has to be initialized in run because the definition of limit ranges
+ * in EPC_CONF as an struct makes it impossible for the linker to understand that in
+ * EPC_CONF this memory zones are signed to the defines in this module.*/
+APP_IFACE_result_e AppIfacePeriodicRegister () {
```
[the remark this answers is not in the document]
Comment (pastorpflores) [reply firmware/Sources/APP/APP_IFACE/app_iface.c:69]: Usually dinos prefer coherence and consistency as key values to respect in architecture definition. But your comment is highly valued.

File: firmware/Sources/APP/APP_SALG/app_salg.c (hunk 1 above)
Comment (pastorpflores) [reply firmware/Sources/APP/APP_SALG/app_salg.c:74]: Meas are used in several modules meanwhile, factors and offsets are only used in MID DABS. Wouldn't it be better is they are in separated structs?

Comment (pastorpflores) [reply firmware/Sources/APP/APP_SALG/app_salg.c]: Understood. I changed MID_REG_meas_s to MID_REG_meas_property_s to create instances of the same type in EPC_CONF with the calibration values. :)

**Review by mariuscrsn — APPROVED** [review]

**Pull request merged**